Net/CMakeLists.txt

```diff
@@ -18,11 +18,11 @@ set_target_properties( Net
 target_link_libraries(Net PUBLIC Poco::Foundation)
 # Windows and WindowsCE need additional libraries
 if(WIN32)
-    target_link_libraries(Net PUBLIC "iphlpapi.lib")
+    target_link_libraries(Net PUBLIC "iphlpapi")
     if(WINCE)
-        target_link_libraries(Net PUBLIC "ws2.lib")
+        target_link_libraries(Net PUBLIC "ws2")
     else()    
-        target_link_libraries(Net PUBLIC "ws2_32.lib")
+        target_link_libraries(Net PUBLIC "ws2_32")
```

Comment:
@bjoe?

Reply:
Well I don't know yet if this will work with microsoft compile. But ws2_32 is better then ws2_32.lib .... I would like to investigate if cmake support .lib suffix for microsoft compiler...